api/include/mfxdefs.h

```diff
@@ -103,6 +103,52 @@ extern "C"
 
 #define MFX_INFINITE 0xFFFFFFFF
 
+#if !defined(MFX_DEPRECATED_OFF) && (MFX_VERSION < MFX_VERSION_NEXT)
+#define MFX_DEPRECATED_OFF
+#endif
+
+#ifndef MFX_DEPRECATED_OFF
```

Comment:
I don't understand how samples got built successfully in our API=latest CI configuration since they should have stepped into warnings, but it seems they did not. Did `-DMFX_DEPRECATED_OFF` in dispatcher influenced samples, tools and totorials build?

Reply:
Samples and tutorials have "-Wno-deprecated-declarations" flag. That's why build doesn't fail.